pgcli/packages/pgspecial/main.py

```diff
@@ -18,6 +18,32 @@
 class CommandNotFound(Exception):
     pass
 
+
+@export
+class PGSpecial(object):
+    def __init__(self):
+        self.timing_enabled = True
+
+        self.commands = COMMANDS.copy()
```

Comment:
I think that means we have to discard the `@special_command` decorator though -- it needs somewhere to store special command metadata